.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_TILECACHE_URL=https://tiles.pacificclimate.org/tilecache/tilecache.py
+REACT_APP_NCWMS_URL=https://services.pacificclimate.org/pcex/ncwms
+REACT_APP_CE_ENSEMBLE_NAME=ce
```

Comment:
Should probable be ce_files.

Reply:
There isn't a `ce` ensemble and hasn't been for a while; the current ensemble is called `ce_files`.

But we set the ensemble via portal routing, so it doesn't actually _matter_ that this environment variable points to a nonexistent ensemble, I think. Possibly this environment variable is overdue to be removed entirely, but I'm not sure if that would have side effects.

Reply:
Right. I've put in an [issue](https://github.com/pacificclimate/climate-explorer-frontend/issues/301).